content/manuals/scout/policy/_index.md

```diff
@@ -146,8 +147,8 @@ The list includes the following vulnerabilities:
 - [CVE-2023-44487 (HTTP/2 Rapid Reset)](https://scout.docker.com/v/CVE-2023-44487)
 - [CVE-2024-3094 (XZ backdoor)](https://scout.docker.com/v/CVE-2024-3094)
 
-You can configure the CVEs included in this list by creating a custom policy.
-Custom configuration options include:
+You can customize this policy to change which CVEs that are considered
+high-profile by configuring the policy. Custom configuration options include:
 
 - **CVEs to avoid**: Specify the CVEs that you want to avoid in your artifacts.
```

Comment:
This is now "Excluded CVEs" and the user lists which CVEs they do _not_ wish to track.

Reply:
Thanks - I will carry this change in #21020